dist/knockout-interpolate.js

```diff
@@ -18,21 +18,42 @@
         return node.nodeType === 3 && node.nodeValue.indexOf("{{") > -1;
     }
 
-    ko.bindingProvider.instance = {
+    function hasInterpolationAttribute(node){
+        return node.attributes && node.attributes["data-koset"];
+    }
+
+    function getValueOfExpression(rawExpression, bindingContext, node){
+        var expression = rawExpression.replace("{{", "").replace("}}", "");
+        // take advantage of existing KO functionality to parse/evaluate binding expression
+        return ko.unwrap(defaultProvider.parseBindingsString("x:" + expression, bindingContext, node).x);
+    }
+
+    function processInterpolationAttribute(node, bindingContext){
+        var bindingValues = defaultProvider.parseBindingsString(node.attributes["data-koset"].value, bindingContext, node);
+        if (bindingValues.hasOwnProperty("visible") && !bindingValues.visible){
+            node.style.display = "none";
+        }
+        if (bindingValues.hasOwnProperty("if") && !bindingValues["if"]){
+            node.innerHTML = "";
+        }
+        if (bindingValues.hasOwnProperty("value")){
+            node.value = bindingValues.value;
+        }
+    }
+
+    ko.bindingProvider.interpolate = {
         nodeHasBindings: function(node) {
-            return hasTextToInterpolate(node) || existingProvider.nodeHasBindings(node);
+            return hasTextToInterpolate(node) || hasInterpolationAttribute(node, this.prefix) || existingProvider.nodeHasBindings(node);
```

Comment:
This line is passing `this.prefix` which is not used in `hasInterpolationAttribute`